ctags/main/parse.c

```diff
@@ -75,8 +75,8 @@ extern void makeSimpleScopedTag (const vString* const name,
 
 		e.kindName = kinds [kind].name;
 		e.kind     = kinds [kind].letter;
-		e.extensionFields.scope[0] = scope;
-		e.extensionFields.scope[1] = scope2;
+		e.extensionFields.scopeKind = &(kinds [kind]);
```

Comment:
Eh, you're right. The whole function will be removed at the end anyway, we use it just for conf but yeah, this is wrong. Will fix.